src/player.cpp

```diff
@@ -9407,6 +9407,14 @@ int player::butcher_factor() const
         item tmp( "toolset", 0 );
         result = tmp.butcher_factor();
     }
+    if ((has_trait("CLAWS")) || (has_trait("CLAWS_RAT")) || (has_active_mutation("CLAWS_RETRACT")) || (has_trait("TALONS"))) {
+        item tmp( "toolset", 0 );
+        result = tmp.butcher_factor();
+    }
+    if ((has_bionic("bio_razor")) || (has_trait("CLAWS_ST"))) {
+        item tmp( "butcher_razor", 0 );
+        result = tmp.butcher_factor();
```

Comment:
This should be `result = std::max( result, tmp.butcher_factor() );` Currently it just overrides the previous butcher factor, which might ave been better. See the lines for inv and weapon below.